test/built-ins/Object/Assign/Override-withStringtarget.js

```diff
@@ -0,0 +1,13 @@
+/*---
+description: Test override of Object.Assign(target,...sources),
+             Every string from sources will be wrapped to objects, and override from the first letter(result[0]) all the time.
+es6id:  19.1.2.1
+author: Wei Guo
+---*/
+
+var target = "ddd";
+var result = Object.assign(target,"aaa","bb2b","1c");
```

Comment:
This test is invalid. String literals will be converted to String instances, and [String instances have non-writable index properties](https://people.mozilla.org/~jorendorff/es6-draft.html#sec-stringgetindexproperty). This, combined with the fact that `Object.assign` invokes the Set internal operation with the *throw* parameter set to **true**, means that we should expect a TypeError here. (As I mentioned below, you can assert this using `assert.throws`.)